deployment/Dockerfile.arm32

```
@@ -0,0 +1,155 @@
+
+# Copy ARM32 binary from builder stage
+COPY --from=builder /usr/src/lavalink-rust/target/armv7-unknown-linux-gnueabihf/release/lavalink-rust /opt/lavalink-rust/
+
+# Copy configuration files
+COPY deployment/config/ /opt/lavalink-rust/config/
+
+# Create plugins directory
+RUN mkdir -p /opt/lavalink-rust/plugins && chown lavalink:lavalink /opt/lavalink-rust/plugins
+
+# Copy plugins if available
+COPY --from=builder /usr/src/lavalink-rust/plugins/*.jar /opt/lavalink-rust/plugins/ 2>/dev/null || true
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)